packages/api-client/src/extensions/localiseConfig.ts

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Really nice functionality. @AdamPawlinski will take a look at it from the code part as well. 

Could you also update the changelog docs with your change? After this PR will be merged I will add you to the contributors of this project as well :)

Reply:
Hey @julienTeam it looks really fine for me. It's updating properly and issues with refreshing manually are not connected with this task. I wonder if you try other properties for strategy, without prefix for default language? 
![vendure-localisation](https://user-images.githubusercontent.com/32042425/151554460-a5e67d65-31b3-4549-9eeb-50934a3af671.gif)

Reply:
> Hey @julienTeam it looks really fine for me. It's updating properly and issues with refreshing manually are not connected with this task. I wonder if you try other properties for strategy, without prefix for default language? ![vendure-localisation](https://user-images.githubusercontent.com/32042425/151554460-a5e67d65-31b3-4549-9eeb-50934a3af671.gif)

Hi Adam, Thank you for your review and comment, 
Indeed, I made quite a lot of test changing the strategy property and in the end, it appears that the no_prefix strategy is probably the best choice.
Also, I is good to add a short code in the LocaleSelector.vue file :  <a :href="switchLocalePath(lang.code)"  **@click:="$i18n.setLocaleCookie(lang.code)"** > to force the cookie setting to the new language selection without reloading and it works well like that.
It's working well on all pages except when you switch language from the Homepage. In this case, you still need to hard reload the page to force the menu to reload whith the correct categories.
I'm still looking for solutions for this.
 Any ideas ?